src/stackdriver-nozzle/config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
```

Comment:
I'd generally prefer flag parsing to happen inside main(). That makes sure anything that needs a flag/config value has to declare it in its method signature, and can't go grabbing initialized package-public vars.